### DISCUSSION TITLE
Relevance Criteria for Usecases

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [SmartDataAnalytics/OpenResearch#111] [Ideas]
For the paper writing meeting tomorrow and the  general discussion about relevance of our usecase i propose to look into:
*    http://wiki.bitplan.com/images/wiki/b/b2/Scenarios.pptx
*    http://wiki.bitplan.com/index.php/Akzeptanz
** Projektion
** Identifikation
 *   https://wiki.tib.eu/confluence/pages/viewpage.action?pageId=125502045

**Comment by WolfgangFahl** [comment]
![grafik](https://user-images.githubusercontent.com/1336221/108346439-a8e29780-71df-11eb-8521-88af7923a9c2.png)

**Reply by WolfgangFahl** [reply]
Plus Kosten / Nutzen KPI

**Comment by WolfgangFahl** [comment]
see also https://en.wikipedia.org/wiki/Kano_model ![Kano Model](https://upload.wikimedia.org/wikipedia/commons/6/68/Kano_model_showing_transition_over_time.png)

**Comment by bschauerte** [comment]
Hi, is there a use case overview with an analysis of user needs and/or workflows? 

"OpenResearch aims at makeing the descriptive metadata on conferences and other formats of scientific events permanently accessible in a high quality through automated processes and scientific data curating. OpenResearch as a sustainable service addresses researchers who search for and publish information on scientific events, as well as universities, information infrastructure institutions, specialised societies, publishers and funding agencies" ... is the best I could find in that direction.
